test/e2e/config/azure-dev.yaml

```diff
@@ -71,4 +74,4 @@ intervals:
   default/wait-machine-remediation: ["30m", "10s"]
   default/wait-deployment: ["5m", "10s"]
   default/wait-job: ["5m", "10s"]
-  default/wait-service: ["3m", "10s"]
+  default/wait-service: ["5m", "10s"]
```

Comment:
I had bumped this to try to fix the issue you found with the etcd cluster not coming online,  Not sure if we want to keep it at 5 mins?